test/python/WMCore_t/WorkQueue_t/DataStructs_t/WorkQueueElement_t.py

```diff
@@ -189,7 +190,10 @@ def testPossibleSites(self):
         """
         # test element ala MonteCarlo
         ele = WorkQueueElement(SiteWhitelist=["T1_IT_CNAF", "T2_DE_DESY"])
-        self.assertEqual(possibleSites(ele), ["T1_IT_CNAF", "T2_DE_DESY"])
+        if PY3:
```

Comment:
We should not really waste time with these, but in the future, I would recommend the following the previous override approach:
```
if PY3:
    self.assertItemsEqual = self.assertCountEqual
```
and replace these `self.assertEqual` testing lists to `self.assertItemsEqual`.

Reply:
> and replace these self.assertEqual testing lists to self.assertItemsEqual

yeah, I know. This is what I should have done, but i felt like that if in py2 we used `assertEqual` than we really wanted to check the order. If you say that we do not really care about the order, then I am happy of using `assertItemsEqual` in all those instances!

Reply:
(changed!)